components/elm/src/biogeochem/DUSTMod.F90

```diff
@@ -25,56 +25,51 @@ module DUSTMod
   use atm2lndType          , only : atm2lnd_type
   use SoilStateType        , only : soilstate_type
   use CanopyStateType      , only : canopystate_type
+  use WaterstateType       , only : waterstate_type
   use FrictionVelocityType , only : frictionvel_type
   use TopounitDataType     , only : top_as
   use LandunitType         , only : lun_pp
   use ColumnType           , only : col_pp
   use ColumnDataType       , only : col_ws
   use VegetationType       , only : veg_pp
-  !
+  !  
   ! !PUBLIC TYPES
   implicit none
   save
   private
   !
   ! !PUBLIC MEMBER FUNCTIONS:
   !
-  public DustEmission   ! Dust mobilization
+  public DustEmission   ! Dust mobilization 
   public DustDryDep     ! Turbulent dry deposition for dust
   !
   ! !PUBLIC DATA:
   !
   real(r8) , allocatable :: ovr_src_snk_mss(:,:)
-  real(r8) , allocatable :: dmt_vwr(:)           ![m] Mass-weighted mean diameter resolved
-  real(r8) , allocatable :: stk_crc(:)           ![frc] Correction to Stokes settling velocity
+  real(r8) , allocatable :: dmt_vwr(:) ![m] Mass-weighted mean diameter resolved
+  real(r8) , allocatable :: stk_crc(:) ![frc] Correction to Stokes settling velocity
   real(r8) tmp1                        !Factor in saltation computation (named as in Charlie's code)
   real(r8) dns_aer                     ![kg m-3] Aerosol density
-  !$acc declare create(ovr_src_snk_mss(:,:))
```

Comment:
Is there a reason to remove these directives? Are these removed as some of these variables are no longer used in the new scheme? Maybe we should remove them in a separate PR.